### ISSUE TITLE
Separate the emby server from the storage server

### ISSUE THREAD

**Issue by WinXaito** [MediaBrowser/Emby#3522]
Hello,

I would like to have an emby server on a VPS, with my media on an external server.

I tried to create a webdav (saber / dav) on the second server and mount a disk on my VPS with davfs2.

But it does not seem to work, emby does not recognize any media. (and I tried with a symlink, but I get a 500 error from the server when reading).

Can you tell me if it's possible to do what I want (separate the emby server from the storage server).

Thanking you.

**Comment by MCTyler** [comment]
You have already waited 3 hours and I have seen numerous questions like this answered on the Forum...Just an idea....https://emby.media/community/

**Comment by YanzheL** [comment]
My solution to this is to mount a [Samba](https://www.samba.org/)/[AFP](https://en.wikipedia.org/wiki/Apple_Filing_Protocol) network share using [GVFS](https://wiki.gnome.org/Projects/gvfs) on linux as a virtual file system.
I did some tests and it works perfectly.
GVFS may be a built-in feature on some linux distributions. My OS is ubuntu 18.04.

### Steps
1. Prepare smb credentials file `smb.key` which contains
```
username
WORKGROUP
password
```
2. Mount SMB share as GVFS volume
replace SMB_HOSTNAME and SHARE_NAME with your info
```shell
gio mount 'smb://SMB_HOSTNAME/SHARE_NAME' < smb.key
```
3. Create a symlink of this volume to your Emby library path
UID is your current linux user id. Emby should have permissions on this volume.
```shell
ln -s '/run/user/UID/gvfs/smb-share:server=SMB_HOSTNAME,share=SHARE_NAME' /path/to/your/emby/library
```

Notes:
1. SMB storage server can running on linux; it is not limited to Windows only.
2. The network connection between SMB server and Emby should be good enough. I suggest to put these VPSs on LAN at same region, eg. Virtual Private Cloud.

**Comment by LukePulverenti** [comment]
Right, we don't have native support for this, I would just mount it in the OS as a local path, then add that to Emby. Does this help?

**Comment by LukePulverenti** [comment]
Also please check our community because we have a lot of users who do this with webdav: https://emby.media/community/index.php?/search/&q=webdav&quick=1

Thanks.
